pcdscalc/be_lens_calcs.py

```diff
@@ -516,13 +516,14 @@ def calc_beam_fwhm(energy, lens_set, distance=None, source_distance=None,
     size_fwhm = gaussian_sigma_to_fwhm(size) / 2.0
 
     if printsummary:
-        print("FWHM at lens   : %.3e" % (fwhm_unfocused))
-        print("waist          : %.3e" % (waist))
-        print("waist FWHM     : %.3e" % (waist * FWHM_SIGMA_CONVERSION / 2.0))
-        print("rayleigh_range : %.3e" % (rayleigh_range))
-        print("focal length   : %.3e" % (focal_length))
-        print("size           : %.3e" % (size))
-        print("size FWHM      : %.3e" % (size_fwhm))
+        logger.info("FWHM at lens   : %.3e" % (fwhm_unfocused))
```

Comment:
```suggestion
        logger.info("FWHM at lens   : %.3e", fwhm_unfocused)
```

And all others

Reply:
ohhh i could not make my mind up with this one, I know we want logger most of the time, but for these prints I want them to be printed to stdout for the user regardless if they have logging level setup - do we still want to use `logging` in this situation too?

Reply:
I'll change them back though - :)

Reply:
Since all the user sessions path through our launcher, you can be sure that logging will always be configured.

Reply:
Ohh I see, that's good then - thank you, changed it back in c0a768d

Reply:
fixed this in c0a768d